frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Pipeline metrics
+    As a user, I want to see an overall how well my Pipeline Runs have gone and how long they take to run.
+
+
+Background:
+    Given user has installed OpenShift Pipelines operator
+    And user has selected namespace "aut-pipelines-metrics"
+    And user is at pipelines page
+
+
+@regression
+Scenario: Pipeline metrics tab   
+    Given pipeline is present 
+    When user clicks on available pipeline
+    Then user is able to see Dashboard tab in pipeline details page
+
+
+@regression
+Scenario: Graphs in dashboard tab
+    Given user is in pipeline details tab
+    And pipeline run is present
+    When user clicks on dashboard tab
```

Comment:
The title `dashboard` is likely to be changed, not confirmed yet. will you create a follow up PR to change it??

Reply:
Sure I'll create a gherkin task for it. Is there a bug/task created for this

Reply:
not yet, but will notify you once it is created.